src/schema-manager/schema-manager-factory.ts

```diff
@@ -27,16 +39,21 @@ export class SchemaManagerFactory {
   create(): SchemaManager {
     const schemaConfig = this._host.getConfig().schema;
     let options: SchemaConfigOptions;
+
     if (typeof schemaConfig === 'string') {
       options = this._convertOptionsFromString(schemaConfig);
     } else {
-      options = schemaConfig;
+      options = schemaConfig as SchemaConfigOptions;
     }
+
     if (isFileType(options)) {
       return new FileSchemaManager(this._host, options.file);
     } else if (isHttpType(options)) {
       return new HttpSchemaManager(this._host, options.http);
+    } else if (isScriptedHttpType(options)) {
```

Comment:
Would you add test cases for this condition to `src/schema-manager/schema-manager-factory.test.ts` ?

Reply:
Added 😄